### ISSUE TITLE
Add Regional drops

### ISSUE THREAD

**Issue by savage13** [zeldamods/objmap-totk#113]
Add Regional drops for Flux, Seafood, Like Like Chest (possibly more)

**Comment by savage13** [comment]
Seafood in ./Pack/Bootup.Nin_NX_NVN/Ecosystem/Ground.ecocat

**Comment by savage13** [comment]
SpObjCapsuleBlockMaster within the Ecocat file? (20% per item)

**Comment by savage13** [comment]
SpObjCapsuleBlockMaster within the Ecocat file? (20% per item)?
Is Weapon in Ecocat file associated with Drop table item HornMaterial - Small Sword and WeaponDefault - SmallSword (for the LikeLIke)?

**Comment by savage13** [comment]
Horn
See Pack/Actor/Enemy_Bokoblin_Dark/GameParameter/GameParameterTable/Bokoblin_Dark.engine__actor__GameParameterTable.json
IsProxySetting set to True, ProxyType is HornMaterial -> grab the drop actor in Ecosystem/HornTypeAndAttachmentMappingTable

**Comment by savage13** [comment]
```  
    4               "ProxyType": "LocalSeaFood",     (See Enemy_Octarock_Snow, Enemy_Octarock_Stone and ecosystem:"Seafood")
    9               "ProxyType": "EcoSystemRain",    (See Obj_LiftRockEldin_A_01, [0x297bfaaad13f6ee5] and ecosystem:"RainBonusMaterial")
   10               "ProxyType": "ZonauBlockMaster", (See Enemy_Zonau_BlockMaster_Senior and ecosystem:"SpObjCapsuleBlockMaster")
  168               "ProxyType": "TBox",             (See Enemy_LikeLike_Crack_RagdollShort)
  388               "ProxyType": "HornMaterial",     (See Enemy_Moriblin_Dark and Horn Data)
11338               "ProxyType": "WeaponDefault",    (See Enemy_LikeLike_Crack_RagdollShort and ecosystem:"Weapon")
```

ecosystem:"" -> `public/game_data/area_data.json`

**Comment by savage13** [comment]
Fixed by commit 4a6ffc0
